### PR TITLE
new demo and example scene in addons

### PR DESCRIPTION
**Pull request by NeoSpark314**
@BastiaanOlij suggested on discord that a sample scene in the addons that would become part of the asset repository would be helpful to get people started (as is done for the OpenVR plugin). 

This is my proposal of a ARVROrigin as a scene with a script that shows some basic usage of the oculus mobile plugin that people could drop into their scene and get oculus_mobile initialized, stick locomotion and some examples on how to access the API and buttons.
I also put the controller .dae mesh in there; but removed the right mesh as it was a broken mirror (flipped triangle order) of the left mesh; I now just mirror it for the right hand.

This PR also contains a proposal to modify the demo scene of this repository. The scene now uses the ARVROrigin scene from the addons folder for interaction and looks like this:

Screenshot from 2019-12-12 21-47-10

### CODE REVIEW

File: demo/Main.gd [hunk 1]
```diff
@@ -1,32 +1,6 @@
-extends Spatial
```
Comment (m4gr3d) [inline demo/Main.gd]: Is this file still needed given there's no active logic?

Comment (NeoSpark314) [reply demo/Main.gd]: I added it to have some place for documentation/comments; but it is not needed at the moment.

**Comment by BastiaanOlij** [comment]
Really awesome work man!
